lib/bambora/client.rb

```diff
@@ -3,31 +3,36 @@
 # Standard Libraries
 require 'base64'
 require 'cgi'
+require 'csv'
 require 'json'
 
 # Gems
 require 'faraday' # HTTP Wraper
 require 'gyoku' # XML Builder
+require 'multiparty' # Multipart/mixed requestss
```

Comment:
Typo: extra "s" at the end.

```suggestion
require 'multiparty' # Multipart/mixed requests
```